src/modules/search-result/author.jsx

```diff
@@ -1,28 +1,80 @@
-import React from 'react'
+import React, { useState, useEffect, useRef } from 'react'
 
 import styles from './author.css'
 
+import { Button } from 'elements'
 import { classNames } from 'utils'
 
-const List = ({ children, className, ...htmlProps }) => (
-  <ol className={classNames.use(styles.list).join(className)} {...htmlProps}>
-    {children}
-  </ol>
+const Item = ({ name, className, ...htmlProps }) => (
+  <li
+    itemProp="author"
+    itemScope=""
+    itemType="https://schema.org/Person"
+    className={classNames.use(className)}
+    {...htmlProps}
+  >
+    <a
+      href={`https://core.ac.uk/search?q=author:(${name})`}
+      className={styles.link}
+    >
+      {name.replace(',', ' ')}
+    </a>
+  </li>
 )
 
-const Item = ({ children, href, ...htmlProps }) => {
-  const ElementType = href != null ? 'a' : 'span'
+const List = ({ authors, className, ...htmlProps }) => {
+  const showMoreRef = useRef(null)
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  useEffect(() => {
+    if (isExpanded)
+      showMoreRef.current.parentElement.nextElementSibling.firstChild.focus()
+  }, [isExpanded])
+
+  if (authors.length <= 3) {
+    return (
+      <ol
+        className={classNames.use(styles.list).join(className)}
+        {...htmlProps}
+      >
+        {authors.map((author) => (
+          <Item name={author.name} />
+        ))}
+      </ol>
+    )
+  }
+
   return (
-    <li
-      itemProp="author"
-      itemScope=""
-      itemType="https://schema.org/Person"
-      {...htmlProps}
-    >
-      <ElementType href={href} itemprop="name">
-        {children}
-      </ElementType>
-    </li>
+    <ol className={classNames.use(styles.list).join(className)} {...htmlProps}>
+      <Item aria-setsize={authors.length} name={authors[0].name} />
+      <Item aria-setsize={authors.length} name={authors[1].name} />
+      <li
+        aria-setsize={authors.length}
+        className={classNames.use(styles.showMore, isExpanded && styles.hide)}
+      >
+        <Button
+          ref={showMoreRef}
+          onFocus={() => {
+            setIsExpanded(true)
+          }}
+          title="Show more authors"
+        >
+          + {authors.length - 3} MORE
```

Comment:
```suggestion
          +{authors.length - 3} more
```

Change the case using `text-transform: uppercase` but keep the text always in the correct case. I would call it design's backward compatibility.

If you want to add a space after `+` use some thinner one, a thin or a hair would be good, or up to your eye. Use [Unicode spaces](http://jkorpela.fi/chars/spaces.html) as a reference.